### ISSUE TITLE
Building a Convolutional Neural Network

### ISSUE THREAD

**Issue by rohit7033** [tensorflow/tensorflow#9880]
### Describe the problem
I just completed the implementation of A Guide to TF Layers: Building a Convolutional Neural Network for the MNIST data set. The training model successfully ran and gave accuracy of 97.3%.

However, the tutorial does not mention how to use this new trained model to supply own images and see the predictions. Does anyone know how to use the output of the training model to make predictions? I see in the tmp/mnist_convnet_model$ folder, there are some output files like  .pbtxt , meta files and index files. But I can't find instructions to use them for making predictions on my own images.

**Comment by hadim** [comment]
GitHub issues are for issues and not to request help. I would strongly suggest you to ask here http://stackoverflow.com/questions/tagged/tensorflow

You might have better chance to get an answer.

**Comment by jubjamie** [comment]
@rohit7033 is this resolved? If not please raise or link a Stack Overflow question, link it here and **close** the issue. I can come and help you if required.

**Comment by concretevitamin** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
